install_plugin_linux.sh

```diff
@@ -17,3 +22,16 @@ else
         echo "Added user_allow_other added to $FUSE_CONF"
     fi
 fi
+
+### Install Cloudfuse
+echo "Installing Cloudfuse"
+apt-get install libssl-dev
+apt-get install ./cloudfuse*.deb
+
+### Install plugin
+echo "Installing Cloudfuse plugin"
+systemctl stop networkoptix-mediaserver.service
+cp cloudfuse_plugin.so /opt/networkoptix-metavms/mediaserver/bin/plugins/
+systemctl start networkoptix-mediaserver.service
```

Comment:
Same here (can we copy and restart, rather than stop-copy-start)?

Reply:
Good point. I changed this to restart.